com.github.powertab.powertabeditor.yml

```diff
@@ -0,0 +1,85 @@
+app-id: com.github.powertab.powertabeditor
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: powertabeditor
+rename-desktop-file: powertabeditor.desktop
+rename-appdata-file: powertabeditor.metainfo.xml
+rename-icon: powertabeditor
+finish-args:
+  # Standard Qt stuff.
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  # For MIDI devices
+  - --device=all
+
+modules:
+  - shared-modules/linux-audio/jack2.json
```

Comment:
JACK needs at least `--filesystem=xdg-run/pipewire-0` in the finish args for audio playback.

Reply:
Good catch, thanks! My testing so far was just with ALSA